### DISCUSSION TITLE
How to change the storage path of a package when i use xrepo install cmd?

### DISCUSSION THREAD

**Discussion by banana16314** [xmake-io/xmake-repo#944] [Q&A]
> Note: If it is a 'bug report' or 'feature request', please select the corresponding issue template, otherwise this issue will be not replied. 

### Describe the problem details
Under the Windows 10 system, I found that the packages installed through xrepo install XXX are in AppData/.xmake in the user directory of the system disk. I want to install the xrepo package to the "libs/" directory I specified.
在windows10系统下，我发现通过xrepo install XXX安装的包都在系统盘用户目录的AppData/.xmake。我想xrepo安装包时安装到我指定的“libs/”目录下。

**Comment by waruqi** [comment]
看文档：https://xmake.io/#/zh-cn/guide/configuration?id=xmake_pkg_installdir
